src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -238,15 +238,16 @@
 
 // Hero
 .pf-c-about-modal-box__hero {
-  display: none; // Don't display the hero at the narrowest breakpoint
-
-  @media only screen and (min-width: $pf-global--breakpoint--sm) {
-    display: block;
-    background-image: var(--pf-c-about-modal-box__hero--sm--BackgroundImage);
-    background-repeat: no-repeat;
-    background-attachment: fixed;
-    background-position: var(--pf-c-about-modal-box__hero--sm--BackgroundPosition);
-    background-size: var(--pf-c-about-modal-box__hero--sm--BackgroundSize);
-    grid-area: hero;
+  display: block;
+  background-image: var(--pf-c-about-modal-box__hero--sm--BackgroundImage);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: var(--pf-c-about-modal-box__hero--sm--BackgroundPosition);
+  background-size: var(--pf-c-about-modal-box__hero--sm--BackgroundSize);
+  grid-area: hero;
+
+  @media only screen and (max-width: $pf-global--breakpoint--sm) {
```

Comment:
Curious why you changed the media query? It makes this PR a breaking change and goes against mobile-first CSS.

Reply:
Initially, I thought that this would reduce one line of code because I wouldn't need to add the line `visibility: visible` to turn back on the visibility. But thanks for catching that this is a breaking change! 
I also updated the wizard description and toolbar action group which did the same thing @mcoker